tests/e2e/suite_test.go

```diff
@@ -46,6 +46,7 @@ const (
 	netDefaultFileName    = "../assets/net-default.xml"
 	numberOfNodesMax      = 30
 	registrationYaml      = "../assets/machineRegistration.yaml"
+	resetMachineInv       = "../assets/reset_machine_inventory.yaml"
```

Comment:
Maybe add `Yaml` in the variable name like the others related yaml variables (`resetMachineYaml` sounds good).